packages/block/src/from-rpc.ts

```diff
@@ -28,8 +30,7 @@ function normalizeTxParams(_txParams: any) {
   // v as raw signature value {0,1}
   // v is the recovery bit and can be either {0,1} or {27,28}.
   // https://ethereum.stackexchange.com/questions/40679/why-the-value-of-v-is-always-either-27-11011-or-28-11100
-  const v: number = txParams.v
-  txParams.v = v < 27 ? v + 27 : v
+  txParams.v = toType(txParams.v, TypeOutput.BigInt)!
```

Comment:
Does this actually work? Why can we now remove the `v < 27 ? v + 27 : v` logic?

Reply:
I see you've updated the test data so that `v`s are now >=27. Does that fit with what we'd expect in a "real" context?

Reply:
I can look at the responses again but Infura/Alchemy appeared to be returning 27 or 28 for the old transaction type and it was breaking at this point.

Reply:
Taking a random transaction (`0x4e9c81762459fd1ecec67d632dbe3fdb0accb33608e300246547692db4087484`) from a recent block on an alchemy node, I'm getting:

``` 
{
	"jsonrpc": "2.0",
	"id": 67,
	"result": {
		"blockHash": "0xa8a73a4d76924e8e6bb32ea6f7416c92abab6fbca701ffeb5e0ccf93915be258",
		"blockNumber": "0xec6807",
		"from": "0x1bc851490add02632f2b41c87a0b8dc3cba4cf0f",
		"gas": "0x4daef",
		"gasPrice": "0x2fa3be745",
		"maxFeePerGas": "0x50f53742f",
		"maxPriorityFeePerGas": "0x77359400",
		"hash": "0x4e9c81762459fd1ecec67d632dbe3fdb0accb33608e300246547692db4087484",
		"input": "0x5f5755290000000000000000000000000000000000000000000000000000000000000080000000000000000000000000a0b86991c6218b36c1d19d4a2e9eb0ce3606eb480000000000000000000000000000000000000000000000000000000011ad50f400000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000000136f6e65496e6368563446656544796e616d69630000000000000000000000000000000000000000000000000000000000000000000000000000000000000007c0000000000000000000000000a0b86991c6218b36c1d19d4a2e9eb0ce3606eb4800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000011ad50f400000000000000000000000000000000000000000000000002788ce887bbefe200000000000000000000000000000000000000000000000000000000000001200000000000000000000000000000000000000000000000000005a3dbe52e10d5000000000000000000000000f326e4de8f66a0bdc0970b79e0924e33c79f1915000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000006887c025200000000000000000000000000f2f400c138f9fb900576263af0bc7fcde2b1b8a800000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000180000000000000000000000000a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48000000000000000000000000eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee000000000000000000000000f2f400c138f9fb900576263af0bc7fcde2b1b8a800000000000000000000000074de5d4fcbf63e00296fd95d33236b97940166310000000000000000000000000000000000000000000000000000000011ad50f4000000000000000000000000000000000000000000000000027e22542580950700000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000004e0000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000030000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000016000000000000000000000000000000000000000000000000000000000000002808000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000064eb5625d9000000000000000000000000a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48000000000000000000000000e68c1d72340aeefe5be76eda63ae2f4bc75141100000000000000000000000000000000000000000000000000000000011ad50f400000000000000000000000000000000000000000000000000000000800000000000000000000000e68c1d72340aeefe5be76eda63ae2f4bc7514110000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000084fe029156000000000000000000000000a0b86991c6218b36c1d19d4a2e9eb0ce3606eb4800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000011ad50f4000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000018414284aab00000000000000000000000000000000000000000000000000000000000000808000000000000000000000000000000000000000000000000000000000000044000000000000000000000000eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee00000000000000000000000000000001000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000064d1660f99000000000000000000000000eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee0000000000000000000000001111111254fb6c44bac0bed2854e76f90643097d00000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000ab4991fe000000000000000000000000000000000000000000000000cc",
		"nonce": "0xae",
		"to": "0x881d40237659c251811cec9c364ef91dc08d300c",
		"transactionIndex": "0xb",
		"value": "0x0",
		"type": "0x2",
		"accessList": [],
		"chainId": "0x1",
		"v": "0x0",
		"r": "0x585a5019401700c284a1e73cb617a1a7b3c7c0f9f30bcf7a3888b49ac8cda983",
		"s": "0x72b03f87e10a1ee05ec90bbcafb35fab253bc918384482b0864e0df53956b4de"
	}
}
```

Reply:
Hmm, maybe I was imagining things then. Let me dig into it again. Now I'm wondering if the 27/28 was only for legacy transactions.

Reply:
Yes, v 27/28 is only legacy.

Reply:
thanks @gabrocheleau that link was informational! since this would depend on chainId too as per my understanding, we should have a helper fn that can handle the range